app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -133,9 +133,15 @@ class FavoriteFragment : Fragment() {
 
         val favFabClickListener: FavoriteFabClickListener = object : FavoriteFabClickListener {
             override fun onClick(event: Event, itemPosition: Int) {
-                favoriteEventViewModel.setFavorite(event.id, !event.favorite)
-                event.favorite = !event.favorite
```

Comment:
Why you remove this logic?

Reply:
I replaced `!event.favorite` with `false` as the statement always marks the favorite property as false. And the second line `event.favorite = !event.favorite` does not really achieve anything as the argument `event: Event` received is just a copy of the event . Please correct me if I'm wrong